app/code/core/Mage/Core/etc/system.xml

```diff
@@ -1256,16 +1256,28 @@
                             <show_in_website>0</show_in_website>
                             <show_in_store>0</show_in_store>
                         </extensions_compatibility_mode>
-                        <crate_admin_user_notification translate="label comment">
+                        <create_admin_user_notification translate="label comment">
```

Comment:
In https://github.com/OpenMage/magento-lts/pull/3721 was decided not to create a breaking change to fix a typo, let's please address the comments made by other users if possible